docs/userguide/distribution.rst

```diff
@@ -25,7 +25,7 @@ the command line ahead of the ``sdist`` or ``bdist`` commands that you want
 to generate a daily build or snapshot for.  See the section below on the
 :ref:`egg_info <egg_info>` command for more details.
 
-(Also, before you release your project, be sure to see the section above on
+(Also, before you release your project, be sure to see the section below on
```

Comment:
Maybe eliminate the directionality altogether and avoid the problems of moving referents.

```suggestion
(Also, before you release your project, be sure to see the section on
```